homeassistant/components/switch/doorbird.py

```diff
@@ -48,31 +47,36 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the DoorBird switch platform."""
-    device = hass.data.get(DOORBIRD_DOMAIN)
+    for doorstation in hass.data.get(DOORBIRD_DOMAIN):
+
+        device = doorstation.device
+
+        switches = []
+        for switch in SWITCHES:
 
-    switches = []
-    for switch in SWITCHES:
-        _LOGGER.debug("Adding DoorBird switch %s", SWITCHES[switch]["name"])
-        switches.append(DoorBirdSwitch(device, switch))
+            _LOGGER.debug("Adding DoorBird switch %s",
+                          SWITCHES[switch]["name"].format(doorstation.name))
+            switches.append(DoorBirdSwitch(device, switch, doorstation.name))
 
-    add_devices(switches)
-    _LOGGER.info("Added DoorBird switches")
+        add_devices(switches)
+        _LOGGER.info("Added DoorBird switches")
```

Comment:
Remove this. There's already a debug log above and the core will log platforms being setup.